src/net.js

```diff
@@ -1,97 +1,107 @@
 
 const Big = require('big.js');
+const july23 = new Date("2023-07-01")
 
-module.exports = function net(gross, nguoiPhuThuoc, region = 1) {
+module.exports = function net(gross, nguoiPhuThuoc = 0, region = 1, date = july23) {
     const grossBig = new Big(gross);
-    console.log("Gross: ", grossBig.toNumber());
-    console.log("Dependents: ", nguoiPhuThuoc);
 
-   
-    var giamThueNguoiPhuThuoc = new Big(0);
-
-    if (nguoiPhuThuoc > 0) {
-
-        giamThueNguoiPhuThuoc = new Big(nguoiPhuThuoc *4400000);
+    let regionMinWage;
+    let grossBasic;
+    // console.log('Date:', date.toISOString());
+
+    switch (true) {
```

Comment:
Switch-case  is to reducing multi if-else.

For this simple checking date if it before july 23, Just use one if is enough. Dont overuse the switch case